### DISCUSSION TITLE
slash in windows makes invalid path

### DISCUSSION THREAD

**Discussion by hf-xz** [nvim-neo-tree/neo-tree.nvim#1103] [Q&A] [answered]
## What am i using

- Windows 11
- git-bash
- [lazyvim](https://github.com/LazyVim/LazyVim)

## What i met

a. when i use `<leader> e`, which means `Explorer NeoTree (root dir)`, it comes out with no item.
b. when i use `<leader> E`, which means `Explorer NeoTree (cwd)`, it works normally.

## What i found

- in case A, the path show on top of the explorer is `D:/Develop/back-end`, linux slash style, this doesn't work.
- in case B, the path is `D:\Develop\back-end`, windows slash style, this work!

## What i want

could there be any method to set the slash style? or some other way to solve the problem?

**Comment by miversen33** [comment]
There have been a handful of windows path specific fixes that have been merged into Neotree (#1077, and #1023 are the most recent).

The events you are describing work "on my machine". It sounds like you are experiencing a bug, in which case a discussion post is not the right place for you. 

To be sure, could you provide more details? What commit of neo-tree are you using (You can find this in Lazy)?

**Reply by hf-xz** [reply]
version 3.2.0
tag     3.2
branch  v3.x
commit  cac2e1b

**Reply by miversen33** [reply, marked as answer]
Thank you :) I will test this specific commit and see if I can recreate your issue. I think I am going to convert this to an issue regardless as it sounds like you are experiencing one :)
